src/components/Simulation/parts.js

```diff
@@ -0,0 +1,38 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { makeStyles } from "@material-ui/styles";
+import { SimulationPart } from "./SimulationPart";
+import { APPBAR_WITH_TABS_HEIGHT } from "../../constants/ui";
+
+const useStyles = makeStyles(theme => ({
+  root: {
+    marginTop: APPBAR_WITH_TABS_HEIGHT,
```

Comment:
I think that if you use the PageContent component you don’t need that anymore. And I just added a fullscreen props to solve the left sidebar margin issue. https://github.com/BLSQ/hesabu-manager/commit/0cfd5b9f95c67792f2d8b81fc2fe7bc3e5fc8284

Reply:
Yes, will change to that